packages/analytics/analytics-metric-provider/src/components/MetricsProvider.vue

```diff
@@ -113,10 +106,6 @@ const timeframe = computed<Timeframe>(() => {
     return props.overrideTimeframe
   }
 
-  if (skuFeatureFlag.value) {
```

Comment:
Pretend this is always true.  (In this case, the function basically becomes `return props.overrideTimeframe || TimePeriods.get(TimeframeKeys.SEVEN_DAY)!`.)